ironfish/src/wallet/wallet.ts

```diff
@@ -29,7 +29,7 @@ export enum TransactionStatus {
   CONFIRMED = 'confirmed',
   EXPIRED = 'expired',
   PENDING = 'pending',
-  UNCONFIRMED = 'unconfirmed',
+  CONFIRMING = 'confirming',
```

Comment:
I think we should stick with the standard names that BTC and other coins use. Which is unconfirmed / confirmed, and pending some times has a different name. They match the balance names too... pending, unconfirmed, confirmed.